test/regression/2007-10-11-illegal-access-after-free-and-branch.c

```diff
@@ -1,21 +1,21 @@
 // RUN: %clang %s -emit-llvm -g -c -o %t1.bc
 // RUN: rm -rf %t.klee-out
-// RUN: %klee --output-dir=%t.klee-out --optimize %t1.bc 2>&1 | FileCheck %s
+// RUN: %klee --output-dir=%t.klee-out --kdalloc --optimize %t1.bc 2>&1 | FileCheck %s
```

Comment:
This test seems explicitly agnostic to the allocator, I don't think we should pick one here.

Reply:
It still depends on KDAlloc to emit the "use after free" message.